src/cairoUtilFuncGen/serialisation.ts

```diff
@@ -48,18 +49,35 @@ function producePackExpression(type: CairoType): (string | Read)[] {
     ];
   }
   if (type instanceof CairoStruct) {
+    if (type.fullStringRepresentation === CairoUint256.fullStringRepresentation) {
+      return [
+        type.name,
+        '{',
+        ...[...type.members.entries()]
+          .flatMap(([memberName, memberType]) => [
+            memberName,
+            ':',
+            'get_u128_try_from_felt_result(',
```

Comment:
Inserting an imported function name without the import is prone to errors. What do you want to accomplish here?

Reply:
Well, warplib uint256 used felt, cairo 1.0 uint256 uses 2x uint128 so it has to felts have to be casted to u128

Reply:
@AlejandroLabourdette made the function `u256_from_felts` but we should start using u128 directly instead in future. But the change imo deserves a separate PR

Reply:
Yeah, I'm thinking we should use `u128_from_felt` builtin function instead and for u256 we can use `u256{u128_from_felt(a), u128_from_felt(b)}`

Reply:
Afaik `u128_from_felt` would be only necessary when transforming literals right now. So we could pretty much get rid of felts I think. In the long term literals will probably also allow to use them as u128 without the conversion so we could get rid of the function then

Reply:
Imo we shouldn't overcomplicate things right now. We've got `u128` literals. In Solidity there is no such type as felt so abi inputs won't provide `felt` as well. We need it now only because the old warp operates on felts

Reply:
The point is imo the function should be removed soon, so is there any sense in considering when to import that?

Reply:
Ok, I was worried that it could be an obscure bug further down the line. But if it won't exists later I agree